Objects/descrobject.c

```diff
@@ -1967,6 +1967,11 @@ static PyObject *
 ga_getitem(PyObject *self, PyObject *item)
 {
     gaobject *alias = (gaobject *)self;
+    // do a lookup for __parameters__ so it gets populated
+    PyObject *params = PyObject_GetAttrString(self, "__parameters__");
+    if (params == NULL) {
+        return NULL;
+    }
```

Comment:
Don't you need to decref params here?

Reply:
Ah, thanks, I originally was using `hasattr`, and forgot to update this.

Reply:
Wait, but if it is NULL, I shouldn't decref it, right?

Reply:
Yeah, I meant on the success branch.

Reply:
Ahh I see, of course.

Reply:
Done, thank you for catching that.